<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>New</class>
 <widget class="QDialog" name="New">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>164</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>467</width>
    <height>164</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>18</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>18</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="label_127">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Saving Location:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="homeBrowseButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>28</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QLineEdit" name="homePathBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QComboBox" name="homeVideoFormat">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Audio Video Interleaved (*.avi)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_131">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>min:sec</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_126">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Project Name:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDateTimeEdit" name="homeRecordingDuration">
     <property name="displayFormat">
      <string>mm:ss</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="homeSaveButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>28</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_128">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Format:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="homeDefaultButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>28</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QLineEdit" name="homePJnameBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_129">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Record Duration:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
